README.md

```diff
@@ -52,15 +57,15 @@ end
 
 You can then start synchronizing files with S3 through ```middleman s3_sync```.
 
-### Configuration Defaults
+#### Configuration Defaults
 
 The following defaults apply to the configuration items:
 
 | Setting                    | Default                            |
 | -----------------          | ----------------------------       |
-| aws_access_key_id          | ```ENV['AWS_ACCESS_KEY_ID']```     |
-| aws_secret_access_key      | ```ENV['AWS_SECRET_ACCESS_KEY']``` |
-| bucket                     | ```ENV['AWS_BUCKET']```            |
+| aws_access_key_id          | -                                  |
```

Comment:
Hmm ok. Well, unless you feel strongly I think it's better to indicate these vars are nil by default, i.e. the user must set them in some way.